changelog/8258.bugfix.md

```diff
@@ -0,0 +1 @@
+Fixed the bug that events from previous conversation sessions would be re-saved in the SQL or Mongo tracker stores when `retrieve_events_from_previous_conversation_sessions` was true. 
```

Comment:
how about adding a link to the tracker store docs?

Reply:
```suggestion
Fixed the bug that events from previous conversation sessions would be re-saved in the [`SQLTrackerStore`](https://rasa.com/docs/rasa/tracker-stores#sqltrackerstore) or [`MongoTrackerStore`](https://rasa.com/docs/rasa/tracker-stores#mongotrackerstore) when `retrieve_events_from_previous_conversation_sessions` was true. 
```

Reply:
let's use the `.mdx` references as in the other changelogs instead of the absolute paths.

Reply:
```suggestion
Fixed the bug that events from previous conversation sessions would be re-saved in the [`SQLTrackerStore`](./tracker-stores.mdx#sqltrackerstore) or [`MongoTrackerStore`](./tracker-stores.mdx#mongotrackerstore) when `retrieve_events_from_previous_conversation_sessions` was true. 
```